js/main.js

```diff
@@ -5,23 +5,73 @@ var $previewImage = document.querySelector('.preview-image');
 var $form = document.getElementById('form');
 var $title = document.getElementById('title');
 var $notes = document.getElementById('notes');
+var $entriesList = document.querySelector('.entries-list');
+var $newEntryBtn = document.querySelector('.new-entry-button');
+var $containers = document.querySelectorAll('.container');
+var $entriesLink = document.querySelector('.entries-link');
 
 function updateEntryImage() {
   $previewImage.src = $photoUrl.value;
 }
 
 function addEntry(event) {
+  event.preventDefault();
   var entries = {
     title: $title.value,
     photoURL: $photoUrl.value,
     notes: $notes.value,
     entryId: data.nextEntryId
   };
-
   data.nextEntryId++;
   data.entries.unshift(entries);
+  $previewImage.src = 'images/placeholder-image-square.jpg';
   $form.reset();
+  $entriesList.prepend(entryList(entries));
+  changeView();
+}
+
+function entryList(entryObj) {
+  var $list = document.createElement('li');
+  $list.setAttribute('class', 'row entry-list');
+
+  var $image = document.createElement('img');
+  $image.setAttribute('class', 'column-half');
+  $image.setAttribute('src', entryObj.photoURL);
+  $image.setAttribute('alt', 'Preview Image');
+  $list.appendChild($image);
+
+  var $div = document.createElement('div');
+  $div.setAttribute('class', 'column-half');
+  $list.appendChild($div);
+
+  var $heading = document.createElement('h2');
+  $heading.textContent = entryObj.title;
+  $div.appendChild($heading);
+
+  var $paragraph = document.createElement('p');
+  $paragraph.textContent = entryObj.notes;
+  $div.appendChild($paragraph);
+
+  return $list;
+}
+
+for (var i = 0; i < data.entries.length; i++) {
+  $entriesList.appendChild(entryList(data.entries[i]));
+}
+
+function changeView(view) {
+  for (var index = 0; index < $containers.length; index++) {
+    var dataView = event.target.getAttribute('data-view');
```

Comment:
You are still using `event` in here, even though you are not using this as an event handler in your submit function. You should create a function that has a `view` parameter that is meant to be the name of the view to switch to as a string, so you can call that function and pass in the view name from `data-view` as an argument.